### PR TITLE
Fixes #15343: Remove rudder-init from documentation

### PR DESCRIPTION
**Pull request by peckpeck**
https://issues.rudder.io/issues/15343

### CODE REVIEW

File: src/reference/modules/installation/pages/_partials/initial_config.adoc [hunk 1]
```diff
@@ -1,42 +1,8 @@
...
-[TIP]
-
-=====
-
-In case of typing error, or if you wish to reconfigure Rudder, you can
-execute this script again as many times as you want.
-
-=====
-
 === Validate the installation
```
Comment (amousset) [inline src/reference/modules/installation/pages/_partials/initial_config.adoc]: Could you add a line in this section advising to go to `Settings -> General -> Allowed Networks` and check and update the list of networks?

**Comment by peckpeck** [comment]
PR updated with a new commit

File: src/reference/modules/installation/pages/_partials/initial_config.adoc [hunk 2]
```diff
@@ -11,6 +11,9 @@ You should see a loading screen, then a login prompt. The default login is:
 
 You are strongly advised to xref:administration:users.adoc#_passwords[change this password] as soon as possible.
 
+Now you should go to Settings -> General -> Allowed Networks and check that the networks listed there peroperly indlude all your nodes network address.
```
Comment (amousset) [inline src/reference/modules/installation/pages/_partials/initial_config.adoc]: properly include

Comment (amousset) [inline src/reference/modules/installation/pages/_partials/initial_config.adoc]: addresses

**Comment by peckpeck** [comment]
PR updated with a new commit

**Pull request merged**